index.html

```diff
@@ -45,19 +44,17 @@
       </div>
       <div ng-view=""></div>
     </div>
+    <div ng-controller="LanguageController as langCtrl" style="position: fixed; left: 60px; bottom: 20px;">
+    <select ng-init="language=langs[0]" ng-options="l as l.name for l in langs" ng-model="language" ng-change="update()">
```

Comment:
This always shows "English" everytime page load or reload.
Maybe need to ordinary iterate it and check currently selected language manually instead of using ng-model. :/